tox.ini

```diff
@@ -1,12 +1,17 @@
 [tox]
 envlist =
     {py27,py36}-django111
+    {py36}-django{200, 210}
     flake8
     docs
 
 [testenv]
 deps =
     django111: Django>=1.11,<2.0
+    django200: Django>=2.0,<2.1
+    django210: Django>=2.1
+    django{111,200,210},docs: git+https://github.com/release-engineering/kobo.git#egg=kobo
```

Comment:
`kobo` is required by Nitrate XMLRPC, not only for building docs. Obviously, it was missed in the `install_requires`. Can you add it to that list instead?